.github/workflows/ccpp.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         export MAKEFLAGS="-j2"
         mkdir build
         cd build
-        cmake .. -DCMAKE_BUILD_TYPE=Debug -DBUILD_WITH_COLLISION_SUPPORT=ON -DPYTHON_EXECUTABLE=$(which python2)
+        cmake .. -DCMAKE_BUILD_TYPE=Debug -DBUILD_WITH_COLLISION_SUPPORT=ON -DPYTHON_EXECUTABLE=$(which python2) -DBUILD_ADVANCED_TESTING=ON
```

Comment:
We should also remove the `-DPYTHON_EXECUTABLE` flag from CI to test how most systems will work by default. This will result in the failure reported upstream in jrl cmake modules.

Reply:
What do you mean? By default, it will choose the Python3 I think or the first one find in the system.

Reply:
Yeah and that's an issue that I've opened upstream. Since that change to the jrl cmake modules we have broken code _unless_ people also explicitly specify the Python version to be the system default. This was a change in the behaviour and requires manual user specification (it auto-configured to the system default before!).

Reply:
You're right. This should only happened for OS with CMake >= 3.12, where Python3 should be the standard. But I can provide a fix to first check for Python2 if nothing is provided. Is the rule you want to get?

Reply:
I will provide the fix.

Reply:
I think I'd like to see something that uses whatever is configured for `python` or the system standard. E.g. Ubuntu 18.04 has CMake >= 3.12 but python default is 2. If someone changes their default it should pick that one up (rather than a switch based on CMake version)